### ISSUE TITLE
Project won't load can't start debugging 

### ISSUE THREAD

**Issue by morrisjoe** [dotnet/roslyn#6743]
Reported by Michal Talaga <mikeon@objectware.pl> and is willing to share his project if we have to follow-up for repro

Project won't load and can't start debugging on some projects.
 
stack trace when debugger crashes:
System.MissingMethodException was unhandled
Message: An unhandled exception of type 'System.MissingMethodException' occurred in Microsoft.CodeAnalysis.Workspaces.dll
Additional information: Method not found: 'Microsoft.CodeAnalysis.Shared.TestHooks.IAsyncToken Microsoft.CodeAnalysis.Shared.TestHooks.IAsynchronousOperationListener.BeginAsyncOperation(System.String, System.Object)'.

**Comment by Pilchie** [comment]
I believe this is a known issue where VS is updated to Update 1, but the corresponding TypeScript update is not installed.  @paulvanbrenk, can you help Michal get fixed up?

**Comment by Exagram** [comment]
I'm having the same VS crash...
VS2015 Version 14.0.24720.00 Update 1
Typescript 1.5.4 (not latest)

However for others on my team it works fine...

**Comment by paulvanbrenk** [comment]
Yes, if this is a TypeScript project the TypeScript 1.7 doesn't seem to be installed. Can you gather the setup log files, using `http://aka.ms/vscollect` and post them somewhere so we can take a look?

After that repairing your VS install should resolve this.

**Comment by praveennet** [comment]
Hi. is VS 2015 Update1 not compatible if we don't have move projects to latest TypeScript (1.7)?

**Comment by Exagram** [comment]
The Typescript 1.5.4 Extension failed with VS 2015 Update 1. I updated it to Typescript 1.7.3 Extension which seems as the only compatible version. Works.

**Comment by paulvanbrenk** [comment]
@praveennet You don't need to upgrade your project, if you have an older compiler installed that will still be used. However the IntelliSense, and Compile On Save, will use the latest version of TypeScript.

**Comment by rido-min** [comment]
I fixed the same error upgrading (manually) to [typescript 1.7](https://visualstudiogallery.msdn.microsoft.com/c7ba4ae7-e161-4e9d-a764-ada948e0e215)

**Comment by jcouv** [comment]
Is this still an issue, or can it be closed?

**Comment by CyrusNajmabadi** [comment]
Closing out.
